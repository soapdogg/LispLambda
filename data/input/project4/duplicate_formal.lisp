(DEFUN DIFF (X X)
    12)