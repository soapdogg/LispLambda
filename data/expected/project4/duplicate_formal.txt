Error! Duplicate formal parameter name!